src/main/java/duke/duke/ui/Ui.java

```diff
@@ -0,0 +1,116 @@
+package duke.ui;
+
+import duke.info.task.Task;
+import duke.utils.Text;
+import duke.info.task.Calendar;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    private final Scanner sc;
+
+    /**
+     * Constructs an Ui object with a new Scanner
+     * that takes input from System.in
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Displays the welcome message specified in
+     * Text.java in the duke.utils package
+     */
+
+    public void showWelcome() {
+        showLine();
+        System.out.println(Text.TEXT_WELCOME);
+        showLine();
+    }
+
+
+    /**
+     * Returns the next line of input from
+     * the Scanner object
+     * @return the String command from I/O
+     */
+
+    public String readCommand() {
+        return this.sc.nextLine();
+    }
+
+    /**
+     * Prints the text divider as specified in
+     * Text.java in the duke.utils package
+     */
+    public void showLine() {
+        System.out.println(Text.TEXT_DIVIDER);
+    }
+
+    /**
+     * Prints the loading error as specified in
+     * Text.java in the duke.utils package
+     */
+    public void showLoadingError() {
+        showLine();
+        System.out.println(Text.TEXT_LOADING_ERROR);
+        showLine();
+    }
+
+    /**
+     * Prints the error message specified by errorMessage
+     * @param errorMessage - the error message to print
+     */
+    public void showError(String errorMessage) {
+        System.out.println(errorMessage);
+    }
+
+    /**
+     * Prints the goodbye message as specified in Text.java
+     * in the duke.utils.package
+     */
+    public void showGoodbye() {
+        System.out.println(Text.TEXT_GOODBYE_MESSAGE);
+    }
+
+    /**
+     * Prints the toString() format of the specified calendar
+     * object
+     * @param calendar - the calendar to be printed
+     */
+    public void showCalendar(Calendar calendar) {
+        System.out.println(calendar.toString());
+    }
+
+    /**
+     * Prints the success message indicating the task was
+     * successfully added to the calendar. Also displays
+     * the total number of tasks in the calendar after the
+     * addition.
+     * @param addedTask - the task that was added
+     * @param numOfTasks - the total number of tasks
+     */
+    public void showTaskAdded(Task addedTask, int numOfTasks) { System.out.println(String.format(Text.TEXT_TASK_ADDED, addedTask, numOfTasks)); }
```

Comment:
Should this be all on a single line? I also noticed the same issue for some methods in this class.